third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -39,20 +41,37 @@ func resourceComputeNetworkPeering() *schema.Resource {
 				ValidateFunc:     validateRegexp(peerNetworkLinkRegex),
 				DiffSuppressFunc: compareSelfLinkRelativePaths,
 			},
+			// The API only accepts true as a value for exchange_subnet_routes or auto_create_routes (of which only one can be set in a valid request).
+			// Also, you can't set auto_create_routes if you use the networkPeering object. auto_create_routes is also deprecated
 			"auto_create_routes": {
-				Type:     schema.TypeBool,
-				ForceNew: true,
-				Optional: true,
-				Default:  true,
+				Type:             schema.TypeBool,
+				Optional:         true,
+				Deprecated:       "auto_create_routes is deprecated use exchange_subnet_routes instead",
```

Comment:
Since we didn't end up adding `exchange_subnet_routes` perhaps just "This field has been deprecated because it's redundant and not user-configurable. It can safely be removed from your config."

Reply:
I adjusted that.